core/storage/user/gae_models_test.py

```diff
@@ -45,6 +45,28 @@ def test_get_by_role(self):
         self.assertEqual(user[0].role, feconf.ROLE_ID_ADMIN)
 
 
+class StoryProgressModelTests(test_utils.GenericTestBase):
+
+    def test_get_multi(self):
+        model = user_models.StoryProgressModel.create(
+            'user_id', 'story_id_1')
+        model.user_id = 'user_id'
+        model.story_id = 'story_id_1'
+        model.put()
+
+        model = user_models.StoryProgressModel.create(
+            'user_id', 'story_id_2')
+        model.user_id = 'user_id'
+        model.story_id = 'story_id_2'
+        model.put()
+
+        story_progress_models = user_models.StoryProgressModel.get_multi(
+            'user_id', ['story_id_1', 'story_id_2'])
+        self.assertEqual(len(story_progress_models), 2)
+        self.assertEqual(story_progress_models[0].user_id, 'user_id')
+        self.assertEqual(story_progress_models[0].user_id, 'user_id')
```

Comment:
Please proofread; these two statements are checking the same thing, and there is far too little validation here. Validate as much of story_progress_models as possible.

Reply:
Done